build.gradle

```diff
@@ -48,6 +49,28 @@ flyway {
     placeholderSuffix = ']'
 }
 
+sourceSets {
```

Comment:
Is this necessary?  I thought Gradle would identify these source sets by default.